debian/changelog

```diff
@@ -1,3 +1,6 @@
+ [Armando Scarpati]
+  * Added a confirmation box to calculation removal via the WebUI
+ 
```

Comment:
Please change this to "Added a confirmation dialog when trying to remove a calculation via the WebUI"